ComponentKit/Core/Scope/CKComponentScopeFrame.mm

```diff
@@ -88,4 +88,40 @@ - (instancetype)initWithHandle:(CKComponentScopeHandle *)handle
   return self;
 }
 
+- (void)injectScopeHierarchy:(_CKChildScopeFrameRet)oldFrame
```

Comment:
Nope, carry on then

> On Aug 24, 2015, at 3:25 PM, Andrew Pouliot <notifications@github.com> wrote:
> 
> In ComponentKit/Core/Scope/CKComponentScopeFrame.mm:
> 
> > @@ -88,4 +88,40 @@ - (instancetype)initWithHandle:(CKComponentScopeHandle *)handle
> >    return self;
> >  }
> >  
> > +- (void)injectScopeHierarchy:(_CKChildScopeFrameRet)oldFrame
> Passing const reference instead of by value is actually less efficient since it can't move in. Do you have a specific reason for this request?
> 
> —
> Reply to this email directly or view it on GitHub.
>